bochs/config.cc

```diff
@@ -3221,6 +3233,21 @@ static int parse_line_formatted(const char *context, int num_params, char *param
     if (parse_port_e9_hack(context, (const char **)(params + 1), num_params - 1) < 0) {
       return -1;
     }
+  } else if (!strcmp(params[0], "iodebug")) {
+#if BX_SUPPORT_IODEBUG
+    if (num_params != 2) {
+      PARSE_ERR(("%s: iodebug directive: wrong # args.", context));
```

Comment:
shouldn't it be the same as port_e9_hack ?
i.e. with enable/disable and all-rings ?
and here it could be parsed with the same method parse_port_e9_hack which will get extra param - path to param tree

Reply:
This Bochs plugin does not operate in that manner. It did not have a prior section in the .bochsrc file. Would you like me to create one? (with enabled= + all rings?)

Reply:
I mean..  @stlintel, this plugin should be "disabled" with:

plugin_ctrl: iodebug=0

Reply:
Agree. Just wanted to avoid code duplication around config.cc